src/controllers/MLFacetQueryController.ts

```diff
@@ -30,6 +34,10 @@ export class MLFacetQueryController {
     this.freezeCurrentValues = true;
   }
 
+  public enableFreezeFacetOrderFlag() {
```

Comment:
If we provide an `enableFreeze...` should we also have a `disableFreeze...`?

Reply:
No you enable it for the request only, these in no case for disabling it